src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getCommentById(id) {
+  return commentsFromServer
+    .filter(comment => comment.postId === id) || null;
```

Comment:
filter always return the array, so null here is redundant

Reply:
And what I must use instead?